pkg/master/master.go

```diff
@@ -586,12 +593,12 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
```

Comment:
I'm tweaking this to take in a

```go
type EndpointReconcilerConfig struct {
  Reconciler EndpointReconciler
  Interval   time.Duration
}
```

and if it's nil, the defaults will be used (default master count reconciler, 10s interval).

Does that sound ok to you?